web/src/containers/admin/ManageAccount.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import { useHistory } from 'react-router-dom';
+
+import { createAccessRequest, completeAccessRequest } from '../../actions/auth';
+
+import StateAccessRequest from '../StateAccessRequest';
+import StateAccessRequestConfirmation from '../StateAccessRequestConfirmation';
+
+const ManageAccount = ({
+  currentAffiliations, 
+  createAccessRequest: createAccessRequestAction
+}) => {
+
+  const history = useHistory();
+
+  const [showConfirmation, setShowConfirmation] = useState(false);
+
+  const handleCreateAccessRequest = async states => {
+    await createAccessRequestAction(states).then(() => {
+      setShowConfirmation(true);
+    });
+  };
+
+  const handleCompleteAccessRequest = () => {
+    completeAccessRequest();
+    history.push('/');
+  };
+
+  return (
+    <div>
+      {showConfirmation && <StateAccessRequestConfirmation action={handleCompleteAccessRequest} />}
```

Comment:
would a ternary operator be more clear here?

showConfirmation ?
 <StateAccessRequestConfirmation ....>
: 
 
<StateAccessRequest ...>